src/compiler/bundle/test/bundle-modules.spec.ts

```diff
@@ -110,11 +110,12 @@ describe('bundle-module', () => {
 
       const r = await c.build();
       expect(r.diagnostics).toEqual([]);
+      console.log(r.filesWritten);
```

Comment:
Is this console.log expected?

Reply:
yep, for now. I am troubleshooting an issue. :)